regen-network/regen-web
# feat: share profile link

## Description

Closes: regen-network/regen-web#2102



Adds the share profile button:
![Screen Shot 2023-09-14 at 4 21 09 PM](https://github.com/regen-network/regen-web/assets/10120306/a7fb97bb-a84c-4d78-8450-95ee593938f8)

For web2 users the link will be like:
- `http://localhost:3000/profiles/<uuid>`

For web3 users the link will be like:
- `http://localhost:3000/profiles/regen1fhjpuh6qh6kjsedtmfxrayvmzlknkzde92ky2j`

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

Using https://deploy-preview-2108--regen-marketplace.netlify.app/

1. Go to your portfolio page, and click the "Copy link to profile" icon.
2. Paste the address in your browser, and observe that you are brought to your public profile page.
3. From your public profile page, click the "Copy link to profile" icon, and observe you are given the public page as well.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
